### ISSUE TITLE
WCAG: Input Assistance

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#342]
# Summary

This review considered:
1. Error identification
2. Labels or Instructions
3. Error Suggestion
4. Error Prevention
5. Redundant Entry
6. Accessible Authentication (minimum)

- revise profile text inputs to remove ambiguity, probably by removing [Name prefix](https://github.com/BirkbeckCTP/hourglass/blob/2c59a5485757c1eb38d4d43e73caf666bae01440/templates/custom/profile.html#L86-L88) once we complete this: https://github.com/BirkbeckCTP/janeway/issues/4137
- help users prevent errors when signing up: https://github.com/BirkbeckCTP/consortial_billing/issues/62

# Audit Results
## Error identification
Validation occurs on form submission.  Error messages are displayed in text form, but there is a problem with timing and context as described in #338 and #333.  This is being addressed there.

## Labels or Instructions
User input is labelled and where appropriate placeholder text is used to show an example of input possible.
For the user profile settings page there some ambiguity between 'Salutation' and 'Name Prefix' Such that for example putting "Dr" in the first won't show up on the profile, but will if in the second.  But it is unclear where these fields are being used, so a user cannot know which to use for what.

### Doesn't show Dr in profile
<img width="515" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/b1a40129-3a7d-4e51-a306-8004ad578c70">

### Does show Dr in profile
<img width="453" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/8894bd68-5c52-4422-ba58-34727e8098f5">

## Error Suggestion
The error for a failed login is appropriate for security in that it notifies of the failure and does not indicate whether an account exists for that user name.

## Error Prevention
The only part of the website that handles legal and/or financial transactions is the supporter signup sheet.  This needs further investigation as to whether it is compliant.

The site does allow login and signup for accounts, and there is validation on this data.  Once logged in, users can access their profile with options to amend this data.

## Redundant Entry

The only information previously provided by a user than must be re-entered is the username and password, which is essential for security. 

## Accessible Authentication (minimum)

email and password inputs are properly marked up as per H100.
Authentication of email for new signups is via email link authentication.

**Comment by joemull** [comment]
I've added two issues to the summary above that will help satisfy these requirements.
